test/integration/sellers-api-client.test.ts

```diff
@@ -56,6 +56,7 @@ describe(`${SellersApiClient.name}`, () => {
             sessionToken: Credentials?.SessionToken || '',
           },
         },
+        roleArn: environment.ROLE_ARN,
```

Comment:
Don't need that here, because we are already assuming the role prior:

```ts
        new AssumeRoleCommand({
          RoleArn: environment.ROLE_ARN,
          RoleSessionName: 'selling-partner-api-axios',
})
```

Which is what the `assumeRoleArn` property does for us under the hood.

What would probably be useful is another integration test that does assume the role by us, and lets axios interceptor to handle that part.